examples/ffi/exporter.cpp

```diff
@@ -127,6 +127,7 @@ int main(int argc, char *argv[]) {
     printf("Response code: %d\n", send_result.http_response.code);
   }
 
+  ddprof_ffi_PushTagResult_drop(tag_result);
```

Comment:
Minor: Perhaps move this to just after the `ddprof_ffi_Vec_tag_push`? It's kinda weird to wait this long to call `drop(tag_result)`.